app/audit/models/ClaimARefundAuditModel.scala

```diff
@@ -38,18 +38,20 @@ case class ClaimARefundAuditModel(balanceDetails: Option[BalanceDetails],
   private def getAvailableCredit: Double = {
     balanceDetails.flatMap(_.availableCredit) match {
       case Some(_) => balanceDetails.get.availableCredit.get.toDouble.abs
-      case (_) => 0.00
+      case _ => 0.00
     }
   }
 
   private def getCreditType(credit: (DocumentDetailWithDueDate, FinancialDetail)): String = {
```

Comment:
can use a case class 'Credit' to make it more readable, this will also avoid this style 'credit._1.' which is also less readable